public/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>Iron Fish Node App</title>
+    <title>Iron Fish — Node App</title>
```

Comment:
For some reason adding dash here make thing work, seem that it triggers some scripts reloading\loading.